facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java

```diff
@@ -294,6 +294,16 @@ private boolean shouldCheckConsistencyConsistency(FacadeOperation facadeOperatio
                     lightblueEntity = getWithTimeout(listenableFuture, methodName, facadeOperation, shouldSource(facadeOperation));
                 }
             } catch (TimeoutException te) {
+
```

Comment:
We should not interrupt write calls, we might interrupt in the middle of  related write operations, leaving corrupt data.